src/client/components/ProfileBlueBar/ProfileBlueBar.styles.css

```diff
@@ -0,0 +1,123 @@
+.profile-bar-container {
+  background-color: #0b5681;
+  width: 14%;
+  min-width: 10em;
```

Comment:
The min-width is needed in this case Rajani, otherwise 14% for the blue bar will be too small when the screen width is too small, and we don't want that. you can test that by removing it

Reply:
@rajanibusani we need this min-width for not to make shrink too much when the screen width is too small.

Reply:
> @rajanibusani we need this min-width for not to make shrink too much when the screen width is too small.
@soujanyapolavarapu yes, soujanya, i am saying that you can keep min-width, but you can remove the width 14%...Did you tested like that?

Reply:
yes dear then it will not be flexible while we change the screen width.for that i added both.